demos/nonlinear_QG_winddrivengyre/qg_refs.bib~

```diff
@@ -0,0 +1,31 @@
+@Book{Pedlosky:1992,
```

Comment:
Sorry about that.  You are correct.

I just pushed it to github.  Do I need to do anything else at this stage?